ui/src/dashboards/components/Dashboard.tsx

```diff
@@ -57,6 +57,7 @@ class DashboardComponent extends PureComponent<Props> {
               timeRange={timeRange}
               manualRefresh={manualRefresh}
               onPositionChange={onPositionChange}
+              {...this.props}
```

Comment:
let's expand this. Using the spread operator leaves it unclear as to what we're actually passing down to the child component

Reply:
Removed